baseapp/baseapp.go

```diff
@@ -124,9 +119,6 @@ type BaseApp struct { // nolint: maligned
 	// if BaseApp is passed to the upgrade keeper's NewKeeper method.
 	appVersion uint64
 
-	// recovery handler for app.runTx method
-	runTxRecoveryMiddleware recoveryMiddleware
```

Comment:
baseapp's `runTxRecoveryMiddleware` (documented in https://docs.cosmos.network/v0.43/core/runtx_middleware.html) is replaced by the `RecoveryTxMiddleware`.

Reply:
Yeah, I'm working for an ADR for this refactor: #9953